components/modal/style/index.tsx

```diff
@@ -297,7 +297,7 @@ const genModalConfirmStyle: GenerateStyle<ModalToken> = (token) => {
 
           [`+ ${confirmComponentCls}-content`]: {
             marginBlockStart: token.marginXS,
-            flexBasis: "100%",
+            flexBasis: '100%',
```

Comment:
```suggestion
            flexBasis: '100%',
            maxWidth: `calc(100% - ${token.modalConfirmIconSize + token.marginSM})`,
```

改这个就好。别动 wordWrap，这个和 4.x 保持一致就行。

Reply:
好嘞，我研究下

Reply:
代码改好了，`changelog`也改好了。
对于`word-break`这个方案不能用的情况我还有些疑惑，并且在历史 pr 里没找到相关的说明，大佬能给些关键词吗

Reply:
break-word 会让英文单词被砍断，不是最合理的排版方式，所以一般不会作为默认样式设置。

Reply:
感谢答疑